models/attribute-group/src/generator.ts

```diff
@@ -0,0 +1,23 @@
+import { ClientLogging } from '@commercetools-test-data/commons';
+import { fake, Generator, sequence } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { TAttributeGroup } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const generator = Generator<TAttributeGroup>({
+  fields: {
+    id: fake((f) => f.string.uuid()),
+    version: sequence(),
+    createdAt: fake(getOlderDate),
+    lastModifiedAt: fake(getNewerDate),
+    name: fake((f) => f.lorem.words(2)),
+    description: fake((f) => f.lorem.sentences(2)),
```

Comment:
These properties are actually `LocalizedString` se we should be returning one of those models.

Reply:
Done in https://github.com/commercetools/test-data/pull/652/commits/ac5cfe5c389dddd194f3880105d4b470b2da6899